packages/vanilla/test/renderers/TableArrayControl.test.tsx

```diff
@@ -73,6 +73,40 @@ const fixture = {
   ]
 };
 
+const fixture2 = {
```

Comment:
Added because editing 'fixture' caused another test to fail, I didn't want to change any other tests.